src/Integration.UnitTests/LocalServices/UnboundProjectFinderTests_SolutionConfigLevel.cs

```diff
@@ -1,218 +0,0 @@
-﻿/*
- * SonarLint for Visual Studio
- * Copyright (C) 2016-2020 SonarSource SA
- * mailto:info AT sonarsource DOT com
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this program; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- */
-
-using System;
-using System.Collections.Generic;
-using System.IO.Abstractions;
-using System.Linq;
-using FluentAssertions;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Moq;
-using SonarLint.VisualStudio.Core;
-using SonarLint.VisualStudio.Core.Binding;
-using SonarLint.VisualStudio.Integration.Binding;
-using SonarLint.VisualStudio.Integration.NewConnectedMode;
-
-// This file contains tests for UnboundProjectFinder that relate to
-// the solution-level config files.
-// The project-level config tests are in another class.
-
-namespace SonarLint.VisualStudio.Integration.UnitTests.LocalServices
-{
-    [TestClass]
-    public class UnboundProjectFinderTests_SolutionConfigLevel
-    {
-        [TestMethod]
-        [DataRow(SonarLintMode.Connected)]
-        [DataRow(SonarLintMode.LegacyConnected)]
-        public void GetUnboundProjects_SolutionBound_EmptyFilteredProjects(SonarLintMode mode)
-        {
-            // Arrange - no projects created
-            var testConfig = new TestConfigurationBuilder(mode, "sqKey1");
-
-            var testSubject = testConfig.CreateTestSubject();
-
-            // Act
-            var result = testSubject.GetUnboundProjects();
-
-            // Assert
-            AssertEmptyResult(result);
-        }
-
-        [TestMethod]
-        [DataRow(SonarLintMode.Connected)]
-        [DataRow(SonarLintMode.LegacyConnected)]
-        public void GetUnboundProjects_ValidSolution_SolutionRuleConfigIsMissing(SonarLintMode mode)
```

Comment:
Deleted, the tests now use mocks and all the relevant testing is part of CSharpVBProjectBinderTests and CFamilyProjectBinderTests.